src/content/docs/alerts/get-notified/muting-rules-suppress-notifications.mdx

```diff
@@ -245,91 +245,117 @@ A muting rule has the following fields and components:
-      </td>
-    </tr>
 
     <tr>
       <td>
-        An issue is activated because of an incident that is **not** muted
+        `updatedAt`
       </td>
 
       <td>
-        Notifications for this issue will be sent.
+        The timestamp when the muting rule was last modified (UTC).
       </td>
     </tr>
 
     <tr>
       <td>
-        An issue is activated due to an incident that <DNT>**is**</DNT> muted
+        `updatedBy`
       </td>
 
       <td>
-        Notifications for this issue will **not** be sent (muted).
+        The user ID of the person who last modified the muting rule.
       </td>
     </tr>
   </tbody>
 </table>
 
-# Muting behavior with workflows [#workflow-behavior]
+## Manage muting rules [#manage]
 
-A triggered incident has a 1:1 ratio with an issue so if an incident is muted then the matching issue will be muted as well.
-Workflows are triggered by issues that can have one or more incidents, therefore there could be a scenario of muted and not muted incidents combined.
+A muting rule condition is the set of individual expressions made up of attributes, operators, and values that define which incidents to target for muting.
 
-Each issue has one of the following muting states:
+You can create, enable, disable, and manage muting rules. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts**</DNT> and click <DNT>**Muting rules**</DNT> on the left navigation pane. Enable or disable muting rules at any time from the <DNT>**Enabled**</DNT> column. You also can edit each rule by clicking the <Icon name="fe-more-horizontal"/> icon on the row of each rule.
 
-* <DNT>**Fully muted (`FULLY_MUTED`)**</DNT>: an issue has all of its open incidents muted (Default value).
-* <DNT>**Partially muted (`PARTIALLY_MUTED`)**</DNT>: an issue that has at least one open incident that is muted and one open incident that is not muted.
-* <DNT>**Not muted (`NOT_MUTED`)**</DNT>: an issue that has no open muted incidents.
+Rules can have one of the following statuses:
 
-For a step-by-step guide on how to set up your workflows, check out an example demo below (approx. 2:17 minutes):
+* <DNT>**Active**</DNT>: Muting is enabled and active.
+* <DNT>**Scheduled**</DNT>: Muting is enabled but not active yet (there's a future schedule).
+* <DNT>**Ended**</DNT>: Muting is enabled, but no longer active (there's no future schedule).
+* <DNT>**Inactive**</DNT>: Muting is disabled.
 
-<Video
-  type="wistia"
-  id="3nf3q2supe"
+<img
+  title="Manage muting rules"
+  alt="Manage muting rules"
+  src="/images/alerts_screenshot-full_muting-rules.webp"
+/>
+
+<figcaption>
+  Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Muting rules**</DNT>: You can create complex muting rules to target a small or large set of unwanted notifications.
+</figcaption>
+
+## Create a muting rule [#create]
+
+<Callout variant="tip">
```

Comment:
Suggestion: If this is a mandatory prerequisite, label with "important" or make a prereq. section